app/src/main/res/values/strings.xml

```diff
@@ -822,7 +822,7 @@
     <!-- Collection and tab deletion prompt dialog message. This will show when the last tab from a collection is deleted -->
     <string name="delete_tab_and_collection_dialog_message">Deleting this tab will delete the entire collection. You can create new collections at any time.</string>
     <!-- Collection and tab deletion prompt dialog title. Placeholder will be replaced with the collection name. This will show when the last tab from a collection is deleted -->
-    <string name="delete_tab_and_collection_dialog_title">Delete collection %1$s?</string>
+    <string name="delete_tab_and_collection_dialog_title">Delete %1$s?</string>
```

Comment:
Did you verify it's okay to overwrite this string id?

Reply:
> Did you verify it's okay to overwrite this string id?

Yes I did, this is the requested change from #12339 , the title for the dialog should be "Delete %1$s?". This is the only place where the string is used, so it should all be good to go.